### PR TITLE
chore: SEC-688 | updates to sca scan action

### PR DESCRIPTION
**Pull request by cypresschris**
- Closes SEC-688

### Additional details

This change addresses an issue where SCA scans inconsistently run.  The changes include implementing the PAT token 
we use for the actions in this repo, as well as performance tweaks to the scan configuration itself.
### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: .github/workflows/snyk_sca_scan.yaml [hunk 1]
```diff
@@ -15,17 +19,26 @@ jobs:
...
+      - name: Run build
+        run: yarn build
       - name: Installing snyk-delta and dependencies
         run: npm i -g snyk-delta
       - uses: snyk/actions/setup@master
       - name: Perform SCA Scan
         continue-on-error: false
         run: |
-          snyk test --all-projects --strict-out-of-sync=false --detection-depth=6 --exclude=system-tests,docker,Dockerfile --severity-threshold=critical
+          snyk test --all-projects --strict-out-of-sync=false --detection-depth=6 --exclude=system-tests,tooling,cli,docker,Dockerfile --severity-threshold=critical
```
Comment (jennifer-shehane) [inline .github/workflows/snyk_sca_scan.yaml]: @cypresschris What's the intention behind excluding the cli here?

Comment (cypresschris) [reply .github/workflows/snyk_sca_scan.yaml]: That CLI project was generating errors at scan runtime along with tooling. -- I thought these were both projects that could be excluded as auxiliary and not part of the release.

Comment (jennifer-shehane) [reply .github/workflows/snyk_sca_scan.yaml]: @cypresschris The CLI has the dependencies that would be of most concern for our users. They are the dependencies that are included in the cypress cli on their system. I don’t think it should be excluded from scans.

Comment (cypresschris) [reply .github/workflows/snyk_sca_scan.yaml]: I've added `CLI` back @jennifer-shehane

**Comment by astone123** [comment]
I'm still seeing that `Snyk_SAST_Scan` has not run on this PR. Is this what we are trying to fix?

**Comment by cypresschris** [comment]
> I'm still seeing that `Snyk_SAST_Scan` has not run on this PR. Is this what we are trying to fix?

Yes

**Review by astone123 — APPROVED** [review]

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**